internal/app/go-filecoin/paymentchannel/manager.go

```diff
@@ -170,10 +170,10 @@ func (pm *Manager) CreatePaymentChannel(clientAddress, minerAddress address.Addr
 
 	err = pm.waiter.Wait(pm.ctx, msgCid, handleResult)
```

Comment:
I believe this Wait is redundant. It waits here, and then go-fil-markets has it wait again (which works because the waiter looks backwards in the chain then waits). Not a problem with this PR obviously. There's no reason to return a message cid that's already been found in the chain.

Reply:
FYI @shannonwells , I'm not confident to remove it.

Reply:
This wait is changed in my current branch to be more like what's  in lotus.